src/index.html

```diff
@@ -2,7 +2,20 @@
 <html>
 
 <head>
-    <title>Ferdi Sahin</title>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="Learn about job openings at Duong and submit your application now">
+    <meta name="keywords" content="data science, software development, jobs">
+    <meta name="author" content="Duong">
+    <meta name="robots" content="index, follow">
+    <meta property="og:title" content="Opening Jobs">
+    <meta property="og:description" content="Learn about job openings at Duong and submit your application now">
+    <meta property="og:image" content="./meta.png">
+    <meta name="twitter:title" content="Opening Jobs">
+    <meta name="twitter:description" content="Learn about job openings at Duong and submit your application now">
+    <meta name="twitter:image" content="./meta.png">
+    <title>Opening Jobs</title>

```

Comment:
Make the title longer please, maybe at least 24 characters.

Reply:
I think it is fine as it is.